src/sentry/replays/endpoints/project_replay_recording_segment_details.py

```diff
@@ -5,25 +5,43 @@
 import sentry_sdk
 from django.http import StreamingHttpResponse
 from django.http.response import HttpResponseBase
+from drf_spectacular.utils import extend_schema
 from rest_framework.request import Request
 
 from sentry import features
 from sentry.api.api_owners import ApiOwner
 from sentry.api.api_publish_status import ApiPublishStatus
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases.project import ProjectEndpoint
+from sentry.apidocs.constants import RESPONSE_BAD_REQUEST, RESPONSE_FORBIDDEN, RESPONSE_NOT_FOUND
+from sentry.apidocs.examples.replay_examples import ReplayExamples
+from sentry.apidocs.parameters import GlobalParams
+from sentry.apidocs.utils import inline_sentry_response_serializer
 from sentry.replays.lib.storage import RecordingSegmentStorageMeta, make_filename
 from sentry.replays.usecases.reader import download_segment, fetch_segment_metadata
 
 
 @region_silo_endpoint
+@extend_schema(tags=["Replays"])
 class ProjectReplayRecordingSegmentDetailsEndpoint(ProjectEndpoint):
     owner = ApiOwner.REPLAY
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PUBLIC,
     }
 
+    @extend_schema(
+        operation_id="Retrieve a replay recording segment",
+        parameters=[GlobalParams.ORG_SLUG, GlobalParams.PROJECT_SLUG],
+        responses={
+            200: inline_sentry_response_serializer("ListReplayRecordingSegments", list[dict]),
```

Comment:
would be great to type the `list[dict]`!

Reply:
Too big!  Its RRWeb so I think I'm going to leave it untyped for now.